fimsauthz-api/v0.yaml

```diff
@@ -0,0 +1,298 @@
+openapi: 3.0.1
+info:
+  title: NUP OAuth2 Authorization Service API Documentation
+  description: |
+    NUP OAuth2 Authorization Service
+  version: v0
```

Comment:
Do we want to change to be like ASTM ; this would be 0.1 etc. 

The URL would still contain  baseURL/v0/resource as per 
https://cloud.google.com/blog/products/gcp/api-design-which-version-of-versioning-is-right-for-you

Reply:
Following ASTM may be reasonable here.

Reply:
Will leave to @issmith1 and @arkits to decide.